feature-libs/organization/account-summary/assets/translations/translations.ts

```diff
@@ -0,0 +1,10 @@
+import { TranslationChunksConfig, TranslationResources } from '@spartacus/core';
+import { en } from './en';
+
+export const accountSummaryTranslations: TranslationResources = {
+  en,
+};
+
+export const accountSummaryTranslationChunksConfig: TranslationChunksConfig = {
+  accountSummary: ['orgAccountSummary'],
```

Comment:
also need to add `breadcrumbs` in the list, right?

Reply:
yes it is in the accountSummary i18n.ts, refactored this part to separate account summary page and account summary list page